packages/open-api-gateway/package.json

```diff
@@ -36,7 +36,8 @@
   },
   "devDependencies": {
     "@aws-prototyping-sdk/nx-monorepo": "^0.x",
-    "@aws-sdk/client-s3": "^3.252.0",
+    "@aws-sdk/client-s3": "^3.6.1",
```

Comment:
Is there a reason for this downgrade? Wonder if it's amplify related as that uses 3.6.1 iirc.

The custom resource uses `transformToString` on the Body returned from an S3 `GetObjectCommand` which was introduced in 3.129.0: https://github.com/aws/aws-sdk-js-v3/blob/main/CHANGELOG.md#31290-2022-07-13

Reply:
this isn't a downgrade, its an upgrade lol

Reply:
`^3.252.0` => `^3.6.1` is a "[downgrade](https://jubianchi.github.io/semver-check/#/^3.252.0/^3.6.1)"